sdk/testing/src/main/java/io/opentelemetry/sdk/testing/junit4/OpenTelemetryRule.java

```diff
@@ -22,23 +27,28 @@
  * tests. This rule cannot be used with {@link org.junit.ClassRule}.
  *
  * <pre>{@code
- * // public class CoolTest {
```

Comment:
While we may not diligently do it everywhere, the `// ` are unfortunately needed to have annotations render properly in Javadoc. Not needed when switching to snippet tag :)